test/functional/rpc_signmessagewithprivkey.py

```diff
@@ -4,27 +4,44 @@
 # file COPYING or http://www.opensource.org/licenses/mit-license.php.
 """Test RPC commands for signing messages with private key."""
 
+from test_framework.descriptors import (
+    descsum_create,
+)
 from test_framework.test_framework import BitcoinTestFramework
 from test_framework.util import (
     assert_equal,
     assert_raises_rpc_error,
 )
 
+
```

Comment:
nit: this extra line can be removed.

Reply:
Note that the extra lines were introduced intentionally, to follow [PEP8](https://peps.python.org/pep-0008/#blank-lines): 
```Surround top-level function and class definitions with two blank lines.```